internal/fs/inode/dir.go

```diff
@@ -64,6 +64,9 @@ type DirInode interface {
 	// true.
 	LookUpChild(ctx context.Context, name string) (*Core, error)
 
+	// Rename the directiory/folder.
+	RenameFolder(ctx context.Context, folderName string, destinationFolderId string) (*gcs.Folder, error)
```

Comment:
can we make them consistent .. folderName string, destinationFolderId string, either folderName, destinationFolderName or folderId, destinationFolderId

Reply:
In the RenameFolder API, the control client currently requires Name and DestinationFolderId. I thought to keep the same structure but as it is part of file system logic we can change it.

Done.